contrib/debian/manpages/bitcoin-cli.1

```diff
@@ -0,0 +1,52 @@
+.TH BITCOIN-CLI "1" "February 2015" "bitcoin-cli 0.10" 
+.SH NAME
+bitcoin-cli \- the remote procedure call client for bitcoin core. 
+.SH SYNOPSIS
+bitcoin-cli [options] <command> [params] \- Send command to Bitcoin Core. 
+.TP
+bitcoin-cli [options] help \- List Bitcoin Core commands.
+.TP
+bitcoin [options] help <command> \- Get help for a command.
+.SH DESCRIPTION
+This manual page documents the bitcoin-cli program. bitcoin-cli is the RPC client used to send commands to Bitcoin Core.
```

Comment:
I would say: "Is *an* RPC client" or even "Is a minimal RPC client". Using "the" seems to imply it's only one.